virtool/config.py

```diff
@@ -159,6 +203,31 @@ def start_server(ctx, host, port, no_check_db, no_check_files, no_client, no_fet
         port=port
     )
 
+    if use_b2c:
+        if config.port != 9950:
+            logger.fatal("registered endpoints for Azure AD B2C require server to run on port 9950")
+            sys.exit(1)
```

Comment:
Here, you are tying implementation in Virtool to the configuration used in our specific tenant. This is not portable. In reality, the registered endpoints will be a FQDN like `https://app.virtool.ca/token`, not at IP + port combination.

Reply:
Yes. Remove those lines permanently. The deployer is expected to configure this all properly.